### ISSUE TITLE
2019, 20, 21, 22 have multi-day hunts

### ISSUE THREAD

**Issue by kaitlyngaynor** [kaitlyngaynor/hopland-hunting#49]
_E-mail to Alison_: Thanks again for cleaning this up. As I peruse the data, I’m wondering if you can clarify the rules and dates of the non-lottery hunt. Is it correct that the hunters are allowed on the property for up to 3 days in a row? 

And is it correct that there was an end-of-season 2-day hunt in 2019 and 2020, and both a start-of-season 3-day hunt AND end-of-season 3-day hunt in 2021 and 2022? In other words, is this correct?

Single-day public (lottery) hunt
2019 - Aug 17, 18, 24, 25
2020 - Aug 8, 9, 15, 16, 22, 23
2021 - Aug 21, 22, 28, 29
2022 - Aug 20, 21, 27, 28

Multi-day trophy hunts
2019 - 9/21 & 9/22
2020 - 9/19 & 9/20
2021 - 8/14, 8/15 & 8/16
2021 - 9/24, 9/25 & 9/26
2022 - 8/13, 8/14 & 8/15
2022 - 9/23, 9/24 & 9/25

I’m trying to figure out whether or not it makes sense to include them along with the other hunters, given that things like “success rates” will start to get biased, and tracks from multiple days are non-independent. But that means losing 29 successful hunters. Going to give it a bit of thought...

**Comment by kaitlyngaynor** [comment]
Another issue with several of these: the fix rates were set to 10 minutes (presumably to save battery), which means that the 3-min interval that we are using for analysis won't work. Another reason to exclude them?

**Comment by kaitlyngaynor** [comment]
Also — on Dropbox, there are only two tracks from the weekend 2021 - 9/24, 9/25 & 9/26 and no tracks from 2022 - 9/23, 9/24 & 9/25. The sheet says that there were 5-6 trackers deployed each day for these 6 days—safe to say that the data just weren’t able to be recovered?

**Comment by kaitlyngaynor** [comment]
I have switched/added dates in green that are different from yours – the only one was we had a beginning season trophy hunt in 2020, but only 2 days.  In 2021, we changed the restricted (trophy) hunt weekend from 2 scouting days and 2 hunting days to 1 scouting day and 3 hunting days per hunter survey results.  Yes, I’m not sure why the iGotUs seem to have just quit on those last September weekends.  They appeared to have plenty of storage left but did not record.  I’m so sorry.  Because of all of these factors, I’m not how accurate it would be to include the restricted hunters, although their patterns may be more similar to the staff hunt patterns? They definitely seemed to have different strategies knowing they had more time.  Just my 2 cents…


Single-day public (lottery) hunt
2019 - Aug 17, 18, 24, 25
2020 - Aug 5, 16, 22, 23
2021 - Aug 21, 22, 28, 29
2022 - Aug 20, 21, 27, 28

Multi-day trophy hunts
2019 - 9/21 & 9/22
2020 – **8/8 & 8/9**; 9/19 & 9/20
2021 - 8/14, 8/15 & 8/16
2021 - 9/24, 9/25 & 9/26
2022 - 8/13, 8/14 & 8/15
2022 - 9/23, 9/24 & 9/25

**Comment by kaitlyngaynor** [comment]
Ok, super, thanks Alison! Again, noooo apologies, just trying to make sure we are making best use of the data we have! It may make sense to just exclude the restricted hunters. We are also excluding the staff hunters, since we presumed they would have different hunting patterns. Although now I’m wondering if it could be interesting to see if they are different, ahhh, just when I thought I was done ;)

**Comment by kaitlyngaynor** [comment]
Update: I looked into including the staff hunters, but we don't have any data on whether or not the staff hunts were successful, so we can set this approach aside. Going to explore any potential differences between the trophy & public hunts.

**Comment by kaitlyngaynor** [comment]
No major differences in cluster, or cluster effects on success. Only difference is in success rates (trophy hunters are more successful).

Added to manuscript:

For the multi-day hunts, we collected loggers at the end of each day, and given that data were collected under the condition of anonymity we were unable to link tracks to a given hunter. We therefore treated each hunter-day as an independent track, and despite issues of pseudoreplication (for n = 31 of 501 tracks), we found no differences in any of our results when we ran the analyses for only the single-day hunts, and therefore feel comfortable that the pseudoreplication is not introducing bias into the study.
